packages/theming/android-theme/src/__tests__/__snapshots__/android-theme.test.ts.snap

```diff
@@ -493,28 +493,28 @@ Object {
     },
     "variants": Object {
       "body1": Object {
-        "face": "Roboto",
+        "face": "\\"Roboto\\"",
```

Comment:
@ankraj12 I'm not sure whether this change was intentional - can you confirm whether this still works for Android?

Reply:
Never mind, looks like same change happened on iOS and it's causing a red box. Will reach out to Travis about this